api/envoy/extensions/filters/http/grpc_json_transcoder/v3/transcoder.proto

```diff
@@ -69,6 +69,19 @@ message GrpcJsonTranscoder {
     bool preserve_proto_field_names = 4;
   }
 
+  message RequestValidationOptions {
+    // When true, rejects HTTP requests with a path that cannot be mapped to a gRPC method.
```

Comment:
By default, a request not defined in the "services" (with a link to the field) will pass-through this filter.
When set to true, the request will be rejected with status 404.

Reply:
Done